_docs/Instructions-Compile.md

```diff
@@ -11,11 +11,14 @@ In order to get set up to work with SSW.SophieBot, you will need the following:
 
 ## Initial Setup
 
-1. [Clone the repo](#step-1-clone-the-repo)
-2. [Create App Settings](#step-2-create-appsettingsjson)
-3. [Open the repo with Composer](#step-3-open-the-repo-with-composer)
-4. [Configure your bot](#step-4-configure-your-bot)
-5. [Test run](#step-5-test-run)
+- [Run with Bot Framework Composer](#run-with-bot-framework-composer)

```

Comment:
I think we should delete

- [Run with Bot Framework Composer](#run-with-bot-framework-composer)

And move the following to the top as the Table of Contents

  - [Prerequisites](#prerequisites)
  - [Initial Setup](#initial-setup)
    - [Step 1. Clone the repo](#step-1-clone-the-repo)
    - [Step 2. Create appsettings.json](#step-2-create-appsettingsjson)
    - [Step 3. Open the repo with Composer](#step-3-open-the-repo-with-composer)
    - [Step 4. Configure your bot](#step-4-configure-your-bot)
    - [Step 5. Test run](#step-5-test-run)

Reply:
Done